### ISSUE TITLE
Enhancement: Don't prompt/confirm when deleting empty files

### ISSUE THREAD

**Issue by brianreavis** [atom/tree-view#693]
This is one subtle, but really nice behavior of Sublime. When creating files sometimes I get the name wrong and, for whatever reason, I'd rather delete the file and re-create than just rename it. Atom adds friction by prompting "Are you sure you want to delete the selected item?" – this gets old. There's no real consequence for deleting an empty file, so the prompt would be great to remove. Adjusting the behavior would also be great when removing files you don't end up needing after scaffolding a project.

TLDR: the prompt gets to be a bit much.

(moved from https://github.com/atom/atom/issues/10313)

**Comment by lee-dohm** [comment]
Upon further review, we would rather not change this behavior. Thanks very much for the suggestion!
